rsc/examination.properties

```diff
@@ -24,7 +24,7 @@ TEMP_STEP = 0.1
 #SAT_UNIT = %
 SAT_INIT = 98
 SAT_MIN = 50
-#SAT_MAX = 100
+SAT_MAX = 100
```

Comment:
This was commented because the Saturation cannot be above 100%, so it is fine to have it hardcoded